src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -1,27 +1,34 @@
 package akka.persistence.cassandra.journal
 
-import java.lang.{ Long => JLong }
+import java.lang.{Long => JLong}
 import java.nio.ByteBuffer
+import java.util.UUID
 
-import com.datastax.driver.core.policies.{LoggingRetryPolicy, RetryPolicy}
-import com.datastax.driver.core.policies.RetryPolicy.RetryDecision
-
-import scala.concurrent._
-import scala.collection.immutable.Seq
 import scala.collection.JavaConversions._
-import scala.math.min
-import scala.util.{Success, Failure, Try}
+import scala.collection.immutable.Seq
+import scala.concurrent._
+import scala.util.{Failure, Success, Try}
 
-import akka.persistence.journal.AsyncWriteJournal
 import akka.persistence._
 import akka.persistence.cassandra._
+import akka.persistence.journal.AsyncWriteJournal
 import akka.serialization.SerializationExtension
-
 import com.datastax.driver.core._
+import com.datastax.driver.core.policies.RetryPolicy.RetryDecision
+import com.datastax.driver.core.policies.{LoggingRetryPolicy, RetryPolicy}
 import com.datastax.driver.core.utils.Bytes
 
 class CassandraJournal extends AsyncWriteJournal with CassandraRecovery with CassandraStatements {
 
+  // TODO: journalId management.
+  // TODO: Uniqueness in distributed environment. Coordination/coordination-less generation?
+  // TODO: Cluster membership change, Journal instances added and removed.
+  // TODO: We need to ensure globally unique journalId. Conflicts would violate the single writer requirement.
+  // TODO: Garbage collecting or infinitely growing journalId set?
+  private[this] val journalId = UUID.randomUUID.toString
```

Comment:
Why is the `journalId` generated and not application-defined? For example, by setting it in `application.conf`?